lib/auth/saml.go

```diff
@@ -462,6 +492,10 @@ func (a *Server) validateSAMLResponse(ctx context.Context, diagCtx *ssoDiagConte
 			vals = append(vals, vv.Value)
 		}
 		log.Debugf("SAML assertion: %q: %q.", key, vals)
+		if key == "remote_server_destination" {
```

Comment:
Ensure the remote_server_destination is present and valid if  requestID == "idemeum_remote_launch"